metrics/metrics.go

```diff
@@ -74,6 +74,7 @@ type Metrics interface {
 	IncErrorsStreaming(routeId string)
 	RegisterHandler(path string, handler *http.ServeMux)
 	UpdateGauge(key string, value float64)
+	ObserveRouteKubeResources(ingress string, routeId string, service string)
```

Comment:
this could be shortened into:
```suggestion
	ObserveRouteKubeResources(ingress, routeId, service string)
```

Reply:
@mlvzk thanks for the review, but there are more serious issues with the current PR.